python/tests/conftest.py

```diff
@@ -100,5 +121,8 @@ def ts():
 
 
 @fixture(scope="session")
-def multi_ts():
-    return [msprime.simulate(i, random_seed=42) for i in range(2, 100, 5)]
+def multi_ts(ts):
```

Comment:
This is a bit vague, as I'd imagine we'll want all sorts of different fixtures like this. What about ``replicate_ts_fixture`` which will return a list of msprime replicate sims?

Reply:
Fixed in https://github.com/tskit-dev/tskit/pull/909/commits/b0b566b26220b9a9214b378c3163b5586b341814